xiplot/assets/dcc_style.css

```diff
@@ -45,7 +45,8 @@
     width: auto;
     border-width: 1px;
     border-style: dashed;
-    border-radius: 5px;
+    border-radius: 8px;
```

Comment:
Why is every CSS style defined twice here with both the concrete and the CSS variable? Is that for some fallback in case there are no CSS variables?

Reply:
I haven't checked if the backwards compatibility is required (it's just something I noticed @TanakaAkihiro was doing)

Reply:
Concrete CSS values were for users who use a browser or a browser version, which doesn't support CSS variables, but it is maybe unnecessary...